pyproject.toml

```diff
@@ -50,7 +50,13 @@ tests = [
     "pytest-asyncio >= 0.23.8",
     "sanic-testing >= 24.6.0"
 ]
-
+telemetry = [
+    "opentelemetry-api >= 1.28.2",
+    "opentelemetry-sdk >= 1.28.2",
+    "opentelemetry-exporter-otlp >= 1.28.2",
+    "opentelemetry.exporter.jaeger.thrift >= 1.21.0",
+    "opentelemetry-instrumentation >= 0.49b2"
+]
```

Comment:
Hey @ctrl-schaff, while running the branch locally, there is the error below. It seems the library just needs to be installed. However, they are here to be installed.

```
ModuleNotFoundError: No module named 'opentelemetry'
```